src/towncrier/_project.py

```diff
@@ -38,47 +38,45 @@ def _get_package(package_dir: str, package: str) -> ModuleType:
 
 
 def get_version(package_dir: str, package: str) -> str:
-    module = _get_package(package_dir, package)
+    version: Any
 
+    module = _get_package(package_dir, package)
     version = getattr(module, "__version__", None)
-
     if not version:
-        raise Exception("No __version__, I don't know how else to look")
+        try:
+            version = importlib_metadata.version(package)
+        except importlib_metadata.PackageNotFoundError:
+            raise Exception(f"Package not installed and no {package}.__version__ found")
 
     if isinstance(version, str):
         return version.strip()
 
-    if isinstance(version, IncrementalVersion):
-        # FIXME:https://github.com/twisted/incremental/issues/81
-        # Incremental uses `.rcN`.
-        # importlib uses `rcN` (without a dot separation).
-        # Here we make incremental work like importlib.
-        return version.base().strip().replace(".rc", "rc")
-
     if isinstance(version, tuple):
         return ".".join(map(str, version)).strip()
 
+    # Try duck-typing as an Incremental version.
+    if hasattr(version, "base"):
+        try:
+            version = str(version.base()).strip()
+            # Incremental uses `X.Y.rcN`.
+            # Standardize on importlib (and PEP440) use of `X.YrcN`:
+            return version.replace(".rc", "rc")  # type: ignore
```

Comment:
FYI this has been fixed on the Incremental side.

Reply:
Not sure what you are referring to. Should anything in this PR be changed as a result?